source/dlangbot/database.d

```diff
@@ -85,8 +85,9 @@ string[][] getContributorsStats()
 
     string[][] ret;
 
-    // first, sum the points of each individual github id
-    foreach(int gid, int points; db.stmt!"
+    version(unittest)
```

Comment:
OK but wouldn't it be easier to accept the date range as a function parameter? Then the unit test would specify the date range it expects, and the live code would use the date range for the current season.

Reply:
I just wanted to avoid hardcoding dates in the unittest. But you are right, it's a much smaller diff this way.